### PR TITLE
Bump prettify dependencies and apply prettification

### PR DESCRIPTION
**Pull request by Razzeee**
This is something we might want to wait with, until we got some PRs merged. As this will cause pretty much every PR to need a rebase.

### CODE REVIEW

**Comment by Razzeee** [comment]
Btw you might want to use https://github.com/mattermost/desktop/pull/255/files?w=0 to review this

**Comment by jasonblais** [comment]
Great! I've added a pending label for now -- as you said, we'll want to wait until more PRs have been merged to v1.4.

I love cleaning up code where possible, so thank you for submitting the PR :1

**Comment by yuya-oc** [comment]
The indent size is defined as `2` in `.editorconfig` and `gulpfile.js`. Why it is changed in actual codes?

**Comment by Razzeee** [comment]
Sorry, I thought those we're changes in the dependencies. Found the problem and will update the PR soon.

**Comment by Razzeee** [comment]
Please have a look again, I hope that it now has the same functionallity

Two changes were needed:
- Use snake_case for options instead of camelCase https://github.com/tarunc/gulp-jsbeautifier/blob/master/CHANGELOG.md#200---2016-03-01
- Use gulp-diff instead of mode/showdiff https://github.com/tarunc/gulp-jsbeautifier#other

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Set our files to use lf, so our prettify will always be happy
```
Comment (Razzeee) [inline .gitattributes]: Fixes the files switching line ending on windows after checkout and therefore showing up as changed after running prettify.

File: gulpfile.js [hunk 2]
```diff
@@ -25,32 +26,38 @@ gulp.task('prettify:verify', ['prettify:sources:verify', 'prettify:jsx:verify'])
 
 var prettify_options = {
   html: {
-    eol: '\n',
-    indentSize: 2
+    indent_size: 2
   },
   css: {
-    eol: '\n',
```
Comment (Razzeee) [inline gulpfile.js]: We can let these eol go, as this is the default value already. So this doesn't change anything.

File: gulpfile.js [hunk 3]
```diff
@@ -248,5 +255,5 @@ gulp.task('sync-meta', function() {
   appPackageJson.description = packageJson.description;
   appPackageJson.author = packageJson.author;
   appPackageJson.license = packageJson.license;
-  fs.writeFileSync('./src/package.json', JSON.stringify(appPackageJson, null, '  ') + '\n');
-});
+  fs.writeFileSync('./src/package.json', JSON.stringify(appPackageJson, null, '  '));
```
Comment (Razzeee) [inline gulpfile.js]: This fixes './src/package.json' changing each time you run prettify or build. As build will add this '\n' and prettify will remove it.

**Comment by Razzeee** [comment]
I was able to address some more nice things that were bothering me, especially when building on windows.

**Comment by Razzeee** [comment]
Btw, as the changelog is much smaller now, this should be able to get merged sooner and not wait for other PRs to go in.

**Comment by yuya-oc** [comment]
Looks nice! Would you add also `"end_with_newline": true` for each option? It would make better.

**Comment by Razzeee** [comment]
@yuya-oc 
changed.
Leads to some changes for html and css files, check the changelog for infos.

**Comment by jasonblais** [comment]
:+1:  Removed `pending` label.

**Comment by yuya-oc** [comment]
Thanks!

**Pull request merged**